hpx/parallel/segmented_algorithms/exclusive_scan.hpp

```diff
@@ -117,25 +117,27 @@ namespace hpx { namespace parallel { inline namespace v1
 
         // sequential segmented OutIter implementation
         template <typename ExPolicy, typename SegIter, typename OutIter,
-            typename T, typename Op>
+            typename T, typename Op, typename Conv>
         static typename util::detail::algorithm_result<ExPolicy, OutIter>::type
         segmented_exclusive_scan_seq(ExPolicy && policy, SegIter first,
-            SegIter last, OutIter dest, T const& init, Op && op, std::true_type)
+            SegIter last, OutIter dest, T const& init, Op && op,
+            std::true_type, Conv && conv)
         {
             typedef hpx::traits::segmented_iterator_traits<OutIter> traits_out;
             return segmented_scan_seq<
                 exclusive_scan<typename traits_out::local_raw_iterator>>(
                 std::forward<ExPolicy>(policy),
                 first, last, dest, init, std::forward<Op>(op),
-                std::true_type());
+                std::true_type(), std::forward<Conv>(conv));
         }
 
         // sequential non segmented OutIter implementation
         template <typename ExPolicy, typename SegIter, typename OutIter,
-            typename T, typename Op>
+            typename T, typename Op, typename Conv>
         static typename util::detail::algorithm_result<ExPolicy, OutIter>::type
         segmented_exclusive_scan_seq(ExPolicy && policy, SegIter first,
-            SegIter last, OutIter dest, T const& init, Op && op, std::false_type)
+            SegIter last, OutIter dest, T const& init, Op && op,
+            std::false_type, Conv && conv)
```

Comment:
This and the below places where Conv is not used are for the non segmented OutIter implementation of segmented InIter inclusive/exclusive scan. @mcopik told me to ignore that for now, but i kept the function signature the same as the other std::true_type template of these functions, which are for the segmented OutIter version.